### ISSUE TITLE
[QUESTION] Can I update the longhorn I have on my rancher via kubectl?

### ISSUE THREAD

**Issue by elton557** [longhorn/longhorn#4780]
Can I update the longhorn I have on my rancher via kubectl? with this script

kubectl apply -f https://raw.githubusercontent.com/longhorn/longhorn/v1.3.2/deploy/longhorn.yaml

**Comment by PhanLe1010** [comment]
No, it is not recommended to mix Rancher UI with kubectl. If you installed Longhorn using Rancher UI, you should continue using Rancher UI to upgrade Longhorn

**Comment by elton557** [comment]
@PhanLe1010 
It's just that since version 2.6.5 I've never been able to update through the interface, only via kubectl

**Comment by PhanLe1010** [comment]
What is the error when you tried to upgrade using the Rancher UI?

**Comment by elton557** [comment]
@PhanLe1010 No error appears, it just doesn't show the upgrade, I don't see how to update, in version 2.5 of rancher it appeared!

**Comment by joshimoo** [comment]
@elton557 is it possible that you installed via the old catalog in rancher 2.5 and now are using rancher 2.6 which only shows the new catalog?  here is a migration guide if that is the case https://longhorn.io/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui/
